### ISSUE TITLE
M

### ISSUE THREAD

**Issue by bonjonrumi** [kgrzybek/modular-monolith-with-ddd#310]

**Comment by Elie-A-98** [comment]
In domain analysis, multiple domains can define their own model for the same real-world entity. This is to better achieve encapsulation and separation of concern. So the UserAccess module has a User model which contains the things releveant within the UserAccess bounded context, and some other modul might also define a User model which has the properties relevant to it's bounded context.

Reference:
https://learn.microsoft.com/en-us/azure/architecture/microservices/model/domain-analysis

![image](https://github.com/kgrzybek/modular-monolith-with-ddd/assets/62180359/5972aeb6-564e-41c6-9cff-aa1af9b45fe0)

**Comment by bonjonrumi** [comment]
Yes but the data is not used in the meetings module. That module is not doing any kind of authentication or uses the login info in any form.
